src/main/resources/lang.yml

```
@@ -0,0 +1,50 @@
+# Mapify Language File
+# This file contains all messages that can be customized by server administrators
+
+# Command messages
+command:
+  no_permission: "&cYou do not have permission to run this command."
+  player_only: "&cThis command can only be used in-game."
+  invalid_url: "&cPlease specify a valid URL."
+  invalid_dimensions: "&cInvalid dimensions, must be in the form WIDTHxHEIGHT: \"1x1\"."
+  dimensions_too_large: "&cDimensions specified are too large."
+  no_image: "&cThis URL does not have an image."
+  not_found: "&cNot file found at URL."
+  unknown_host: "&cUnknown host '%s'. This could be because it's invalid, or the server does not have access to it. Perhaps try a different URL."
+  not_https: "&cThis URL is not https. You must use a URL with https."
+  domain_not_whitelisted: "&cThis is not a valid domain, please contact your server administrators if you believe this is an issue."
+  domain_not_whitelisted_op: "&cThis domain is not whitelisted in your Mapify config."
+  domain_not_whitelisted_op_suggestion: "&cRun &4/mapify config whitelist add %s &cto automatically add it to the config."
```

NOTE: The op message what split onto two lines for clarity, they are part of the same message.
```suggestion
command.domain.not-whitelisted: "&cThis is not a valid domain, please contact your server administrators if you believe this is an issue."
command.domain.not-whitelisted.op: "&cThis domain is not whitelisted in your Mapify config.\n&cRun &4/mapify config whitelist add %s &cto automatically add it to the config."
```